### DISCUSSION TITLE
Subscribtion

### DISCUSSION THREAD

**Discussion by MrLight223** [d4rken-org/sdmaid-se#1481] [Q&A] [answered]
How can i cancel my subscribtion?

**Comment by MarkyMarkDE** [comment, marked as answer]
In Playstore over your subscriptions option
On github over the sponsoring link from darken and then "manage sponsorship" or similar named

**Comment by NurMiApp** [comment]
I'm sorry Matthias was bad. We are going back SD Maid the triage feature request where is it located.

Regards, the Nurkhametov Mike.
